Added docs for enable_mdns option (#1118)
OttoWinter/esphomedocs: guides/faq.rst

    Alternatively, you can make esphome use ICMP pings to check the status of the device
    with the Hass.io Addon ``"status_use_ping": true,`` option or with docker ``-e ESPHOME_DASHBOARD_USE_PING=true``
    See also https://github.com/esphome/issues/issues/641#issuecomment-534156628.
    
.. _faq-notes_on_disabling_mdns:

Notes on disabling mDNS
------------------------------------------------------------------------------
Some of ESPHome's functionalities rely on mDNS, so naturally :ref:`disabling <wifi-configuration_variables>` it will cause these features to stop working.
Generally speaking, disabling mDNS without setting a :ref:`static IP address <wifi-manual_ip>` (or a static DHCP lease) is bound to cause problems. This is due to the fact that mDNS is used to find the IP address of each ESPHome nodes.

- You will not be able to use the node's hostname to ping, find it's IP address or connect to it.

> - Automatic discovery in Hass.io when using :doc:`native API </components/api>` relies on mDNS broadcast messages to detect presense of new ESPHome nodes. If you need to use the native API with mDNS disabled, then you will have to use a static IP address AND manually add that IP address to Hass.io as an ESPHome integration.

Reviewer: ```suggestion
- Automatic discovery in Home Assistant when using :doc:`native API </components/api>` relies on mDNS broadcast messages to detect presence of new ESPHome nodes. If you need to use the native API with mDNS disabled, then you will have to use a static IP address and manually add the ESPHome integration with the fixed IP address.
```
